src/spartan/polys/eq.rs

```diff
@@ -75,16 +95,20 @@ mod tests {
   use pasta_curves::Fp;
 
   fn test_eq_polynomial_with<F: PrimeField>() {
-    let eq_poly = EqPolynomial::<F>::new(vec![F::ONE, F::ZERO, F::ONE]);
+    // Most-significant 'bit' first.
+    let eq_poly = EqPolynomial::<F>::new(vec![F::ONE, F::ZERO, F::ZERO]);
```

Comment:
Yes, good change: testing this with palindromic MLEs is pointless and already bit me once.